models/Posting.js

```diff
@@ -29,14 +29,17 @@ const PostingSchema = Schema({
     type: [String],
   },
   requestedItems: [String],
-  // imgSrc: {
-  //   data: Buffer,
-  //   contentType: String,
-  // },
+  images: [String],
   ownerId: {
     type: String,
     required: true,
   },
+  ownerUsername: String, 
+  location: String,
```

Comment:
location as in address? should we keep this on the user level as well?

Reply:
The location is currently our `city, province`. Unless we completely forgo including it, I don't think we'd be able to keep it with the user only. For our search results, the location is listed on the preview. If we were to keep it separate from the posting, the frontend would need to do an api call for each posting to get the user information. Unless you want to handle all of that on the backend